cordova/index.html

```diff
@@ -7,13 +7,13 @@
 <meta name="generator" content="Jekyll v4.0.0" />
 <meta property="og:title" content="Scanbot SDK Cordova Plugin" />
 <meta property="og:locale" content="en_US" />
-<link rel="canonical" href="https://scanbotsdk.github.io/documentation/cordova/" />
-<meta property="og:url" content="https://scanbotsdk.github.io/documentation/cordova/" />
+<link rel="canonical" href="http://localhost:4000/documentation/cordova/" />
+<meta property="og:url" content="http://localhost:4000/documentation/cordova/" />
 <meta property="og:site_name" content="Scanbot SDK Cordova Plugin" />
 <script type="application/ld+json">
-{"url":"https://scanbotsdk.github.io/documentation/cordova/","name":"Scanbot SDK Cordova Plugin","headline":"Scanbot SDK Cordova Plugin","@type":"WebSite","@context":"https://schema.org"}</script>
+{"url":"http://localhost:4000/documentation/cordova/","headline":"Scanbot SDK Cordova Plugin","name":"Scanbot SDK Cordova Plugin","@type":"WebSite","@context":"https://schema.org"}</script>
```

Comment:
`http://localhost:4000` is wrong. Pls make sure to use the script `build.sh` for production.
(`dev-serve.sh` is for local dev/test purposes only)